cmd/pw_router/main.go

```diff
@@ -120,7 +142,7 @@ func main() {
 		&cli.IntFlag{
 			Name:    "num-workers",
 			Usage:   "Number of workers to process updates.",
-			Value:   4,
+			Value:   10,
```

Comment:
because Nats.io doesn't do any internal buffering itself - you have to be able to process all the messages it sends you.

so we have a buffer of messages and enough workers to deal with a spike

Reply:
I assume that's what the JetStream part of NATs does? Do we either prescribe the use of it, buffer on our side or over-commit workers. I guess a default of 10 does the most later. I think Jetstream in prod is most likely the answer... but I suspect we'd need some experience running it to know for sure.

Reply:
I currently am not making use of Jetstream AFAIK, this is just a basic implementation